src/view-backend.c

```diff
@@ -103,7 +103,7 @@ wpe_view_backend_set_input_client(struct wpe_view_backend* backend, const struct
 void
 wpe_view_backend_set_fullscreen_client(struct wpe_view_backend* backend, const struct wpe_view_backend_fullscreen_client* client, void* client_data)
 {
-    assert(!backend->fullscreen_client);
+    assert(!backend->fullscreen_client || !client);
```

Comment:
I would either remove the assert, or else parallel it in set_backend_client() and set_input_client(). Don't see any reason the fullscreen client should be different from the other two.

Reply:
Ok, removed the assert.